OpenLaTeXStudio/EditorModule/src/latexstudio/editor/settings/ApplicationSettings.java

```diff
@@ -26,13 +32,17 @@
     private static final String LATEX_PATH      = "latex.path";
     public static final String AUTOCOMPLETE_DELAY = "autocomplete.delay";
     public static final String LINEWRAP_STATUS = "linewrap.status";
-    public static final String AUTOCOMPLETE_STATUS = "autocomplete.status";

```

Comment:
This was replaced by the new mechanism, which defines the string in the ApplicationSettings.Settings enum.